docs/proposals/PROPOSAL_H.md

```diff
@@ -0,0 +1,271 @@
+    "mediaType": "application/vnd.oci.empty.v1+json",
+    "digest": "sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a",
+    "size": 2
+  },
+  "layers": [
+    {
+      "mediaType": "application/nfd.image-compatibility.spec.v1alpha1+json",
+      "digest": "sha256:4a47f8ae4c713906618413cb9795824d09eeadf948729e213a1ba11a1e31d052",
+      "size": 1710,
+      "annotations": {
+        "name": "profileH"
+      }
+    }
+  ],
+  "annotations": {
+    "oci.opencontainers.image.created": "2024-03-27T08:08:08Z"
+  }
+}
+```
+
+The new artifact type for image compatibility MUST be referenced in OCI documentation.
+
+The schema version is to use string semver instead of int version, to help tool in content deserialization.
+
+The media type is defined as nfd.image-compatibility.spec, in order to be defined agnostic to cluster management software.
+
+## image compatibility spec
+
+The spec structure is quite simple, a flat json file. NFD means the Kubernetes node feature discovering SIG.
+
+Schema:
+
+- **`compatibilities`** *list of compatibility set*
+
+  This REQUIRED property describes all image compatibility sets, contains at least one compatibility set.
+
+  - **`compatibility set`** *list of compatibility requirement*
+
+    This REQUIRED property describes compatibility requirements in one compatibility set.
+
+    If one of the compatibility set passes the compatibility check against the target node, that means the node is compatibile with the image.
+
+    There are three category key-value pair inside the compatibility set.
+
+    - **`compatibility feature field`**
+
+      This REQUIRED property describes a feature compatibility requirement on the target host, the field is named with NFD feature label.
+
+      The value of compatibility field in the compatibility artifact SHOULD be described in contiguous or discontiguous range if the value discovered on nodes can be different.
+
+      It's the tool's responsibility to intepret the value and matching critriea with the value discovered from the node via NFD.
+
+      Inevitablely, some compatibility fields MAY be duplicated among compatibility sets.
+
+      The value type of this field is STRING.
+
+    - **`compatibility score field`**
+
+      This OPTIONAL property describes the score of the compatibility set.
+
+      Only one compatibility score filed in one compatibility set.
+
+      The highest score of the compatibility set means this compatibility set is the most optimal one.
+
+      If this field is not present, the score of this compatibility set is 0.
+
+      If compatibility sets have same score, that means any of these compatibility sets pass the compatibility check is successful in matching the compatibility requirements.
+
+      The compatibility-score is used for selecting optimal image in descending order for those compaibility sets pass the comaptibility check.
+
+      The value type of this field is INT.
+
+    - **`annotation field`**
```

Comment:
Why do we need a separate annotation field? Couldn't one of the key value pairs just be used for this purpose?

Reply:
the author can comment on each compatibility set to provide some human readble information. If there is no annotation inside compatibility set, if there is long list of compatibility fields, hard to figure out what's the difference for different compatibility set.

Reply:
I am still feeling this is extra.

Reply:
If we decide to keep the list of compatibilities in the final proposal within single artifact then I think it can be useful. 
I would probably change the name from `annotation` to `description`.

Reply:
>I would probably change the name from annotation to description.

ok, change it to description.

too many annotations in specs